pkgs/tools/security/gnupg/21.nix

```diff
@@ -23,9 +23,14 @@ stdenv.mkDerivation rec {
   };
 
   buildInputs = [
-    pkgconfig libgcrypt libassuan libksba libiconv npth gettext texinfo
-    readline libusb gnutls adns openldap zlib bzip2
-  ];
+    pkgconfig libgcrypt libassuan libksba libiconv npth gettext texinfo ]
+      ++ optional (adns     != null) adns
```

Comment:
> If you are developing on a local nixpkgs copy and you trigger the build of all the dependencies before you get to build yours - that takes a lot of time.

What dependencies? If you pass null instead of the real package, then obviously the package won't be built, no?